### PR TITLE
zh-translation: .\docs\root\configuration\listeners\network_filters\e…

### PR DESCRIPTION
**Pull request by vgbhfive**
…xt_authz_filter.rst

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes: .\docs\root\configuration\listeners\network_filters\ext_authz_filter.rst
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] #330
[Optional Deprecated:]

### CODE REVIEW

**Review by majinghe — APPROVED** [review]

**Comment by majinghe** [comment]
@vgbhfive 非常棒的翻译。

**Comment by majinghe** [comment]
@WisWang 看一下这个 PR。

File: docs/root/configuration/listeners/network_filters/ext_authz_filter.rst [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
 
-The external authorization network filter calls an external authorization service to check if the
-incoming request is authorized or not. If the request is deemed unauthorized by the network filter
-then the connection will be closed.
+外部授权网络过滤器调用外部授权服务去检查传入请求是否被授权。如果请求被网络过滤器视为未经授权那么连接将会被关闭。
 
 .. tip::
-  It is recommended that this filter is configured first in the filter chain so that requests are
-  authorized prior to rest of the filters processing the request.
+  建议在过滤器链中首先配置此过滤器，以便请求可以在其他过滤器处理请求之前对其进行授权。
```
Comment (WisWang) [inline docs/root/configuration/listeners/network_filters/ext_authz_filter.rst]:
```suggestion
  建议在过滤器链中首先配置此过滤器，以便可以在其他过滤器处理请求之前授权请求。
```

File: docs/root/configuration/listeners/network_filters/ext_authz_filter.rst [hunk 2]
```diff
@@ -54,30 +49,26 @@ A sample filter configuration could be:
...
-  ok, Counter, Total responses from the authorization service that were to allow the traffic.
-  cx_closed, Counter, Total connections that were closed.
-  active, Gauge, Total currently active requests in transit to the authorization service.
-
-Dynamic Metadata
+  total, Counter, 来自过滤器的响应总数
+  error, Counter, 连接外部服务的异常总数
+  denied, Counter, 来自授权服务的拒绝的响应总数
+  disabled, Counter, 由于过滤器未启用而未调用外部服务的通过请求总数
+  failure_mode_allowed, Counter, 由于 failure_mode_allow 设置未 true 而允许的请求异常总数
```
Comment (WisWang) [inline docs/root/configuration/listeners/network_filters/ext_authz_filter.rst]:
```suggestion
  failure_mode_allowed, Counter, 由于 failure_mode_allow 设置为 true 而允许的请求异常总数
```

File: docs/root/configuration/listeners/network_filters/ext_authz_filter.rst [hunk 3]
```diff
@@ -54,30 +49,26 @@ A sample filter configuration could be:
...
+
+动态元数据
 ----------------
 .. _config_network_filters_ext_authz_dynamic_metadata:
 
-The External Authorization filter emits dynamic metadata as an opaque ``google.protobuf.Struct``
-*only* when the gRPC authorization server returns a :ref:`CheckResponse
-<envoy_v3_api_msg_service.auth.v3.CheckResponse>` with a filled :ref:`dynamic_metadata
-<envoy_v3_api_field_service.auth.v3.CheckResponse.dynamic_metadata>` field.
+外部授权过滤器仅在 gRPC 授权服务返回一个 :ref:`检查响应 <envoy_v3_api_msg_service.auth.v3.CheckResponse>` 并包含 :ref:`dynamic_metadata <envoy_v3_api_field_service.auth.v3.CheckResponse.dynamic_metadata>` 字段时才会将动态元数据作为一个不透明的 ``google.protobuf.Struct`` 。
```
Comment (WisWang) [inline docs/root/configuration/listeners/network_filters/ext_authz_filter.rst]:
```suggestion
只有当 gRPC 授权服务返回一个包含 :ref:`dynamic_metadata <envoy_v3_api_field_service.auth.v3.CheckResponse.dynamic_metadata>` 字段的 :ref:`CheckResponse <envoy_v3_api_msg_service.auth.v3.CheckResponse>` 时，外部授权过滤器才会将动态元数据以不透明 ``google.protobuf.Struct`` 的形式发出。
```

**Review by WisWang — COMMENTED** [review]
提了一点儿小建议  o(*￣︶￣*)o

**Comment by majinghe** [comment]
@WisWang 看一下这个 PR 是否还有问题。

**Review by WisWang — APPROVED** [review]

**Pull request merged**